rethinkdb/README.md

```diff
@@ -0,0 +1,77 @@
+# Agent Check: RethinkDB
+
+## Overview
+
+[RethinkDB][1] is a distributed documented-oriented NoSQL database, with first class support for realtime change feeds.
+
+This check monitors a RethinkDB cluster through the Datadog Agent and collects metrics about performance, data availability, cluster configuration, and more.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+The RethinkDB check is included in the [Datadog Agent][3] package. No additional installation is needed on your server.
+
+### Configuration
+
+1. Recommended: add a `datadog-agent` user with read-only permissions on the `rethinkdb` database. Use the following ReQL commands, referring to [Permissions and user accounts][4] for details:
```

Comment:
nit: is it the only approach possible? i think we can remove the `recommended` wording here

Reply:
Technically users could use the `admin` account (with or without a password, depending on their setup), but it has read/write permissions over everything. So it's definitely better that they setup a dedicated read-only user. Which means it's probably okay to make this a mandatory step, yes… Will drop the `Recommended` bit.